corehq/apps/receiverwrapper/views.py

```diff
@@ -160,17 +160,15 @@ def _submission_error(request, message, count_metric, metric_tags,
     return response
 
 
-def _record_metrics(tags, submission_type, response, result=None, timer=None):
+def _record_metrics(tags, submission_type, response, timer=None):
     tags += [
         'submission_type:{}'.format(submission_type),
         'status_code:{}'.format(response.status_code)
     ]
 
-    if response.status_code == 201 and timer and result:
+    if response.status_code == 201 and timer:
         tags += [
             'duration:%s' % bucket_value(timer.duration, (1, 5, 20, 60, 120, 300, 600), 's'),
-            'case_count:%s' % bucket_value(len(result.cases), (2, 5, 10)),
-            'ledger_count:%s' % bucket_value(len(result.ledgers), (2, 5, 10)),
```

Comment:
we now have better versions of these in `commcare.load.case` and `commcare.load.ledger`